python/ray/tune/log_sync.py

```diff
@@ -66,14 +66,17 @@ class _LogSyncer(object):
     This syncs files from workers to the local node, and optionally also from
     the local node to a remote directory (e.g. S3)."""
 
-    def __init__(self, local_dir, remote_dir=None):
+    def __init__(self, local_dir, remote_dir=None, verbose=True):
         self.local_dir = local_dir
         self.remote_dir = remote_dir
         self.last_sync_time = 0
         self.sync_process = None
         self.local_ip = ray.services.get_node_ip_address()
         self.worker_ip = None
-        print("Created LogSyncer for {} -> {}".format(local_dir, remote_dir))
+        self.verbose = verbose
+        if self.verbose:
```

Comment:
Let's not use ad-hoc log flags, either leave it as is, or switch to using logging.